include/envoy/local_info/local_info.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "envoy/common/pure.h"
+
+namespace LocalInfo {
+
+/**
+ * Information about the local environment.
+ */
+class LocalInfo {
+public:
+  virtual ~LocalInfo() {}
+
+  /**
+   * Human readable network address.
```

Comment:
nit: can you put also example for this. Human readable network address, e.g., ...